connector/src/main/java/org/geysermc/connector/network/translators/java/world/JavaBlockChangeTranslator.java

```diff
@@ -46,7 +47,7 @@ public void translate(ServerBlockChangePacket packet, GeyserSession session) {
         Position pos = packet.getRecord().getPosition();
         boolean updatePlacement = !(session.getConnector().getConfig().isCacheChunks() && session.getConnector().getWorldManager().getBlockAt(session, pos.getX(), pos.getY(), pos.getZ()).getId() == packet.getRecord().getBlock().getId());
         ChunkUtils.updateBlock(session, packet.getRecord().getBlock(), packet.getRecord().getPosition());
-        if (updatePlacement) {
+        if (updatePlacement && session.getConnector().getPlatformType() != PlatformType.BUKKIT) {
```

Comment:
I'm not really fond that we have to hardcode `PlatformType.BUKKIT` here, so any alternatives are more than welcome.

Reply:
I think it's fine, but in the future if you think you can find a better way to do this, please feel free to include that in a separate PR.